wealth-watch-script.sql

```diff
@@ -0,0 +1,35 @@
+create table user
+(
+id int unsigned not null unique auto_increment,
+name varchar(45) not null unique,
```

Comment:
Let's change this to `varchar (60)` and also add some new columns 
```mysql
email VARCHAR(200) NOT NULL UNIQUE,
created DATETIME NOT NULL,
updated DATETIME NOT NULL
```

Add the created and updated columns to every table not only the user here.

Reply:
done.